site/content/en/docs/administration/advanced/upgrade_guide.md

```diff
@@ -56,6 +56,52 @@ To upgrade CVAT, follow these steps:
   docker logs cvat_server -f
   ```
 
+## Upgrade CVAT after v2.25.0
+
+In version 2.25.0, CVAT changed the location where the export cache is stored.
+The following Python script can be used to remove outdated files from the previous location:
+
+```python
+import shutil
+from pathlib import Path
+from django.utils import timezone
+from tqdm import tqdm
```

Comment:
`tqdm` is not included in the server dependencies, so I don't think you should use it here.